new/install-ohai-plugins-at-bootstrap.md

```diff
@@ -0,0 +1,51 @@
+---
+RFC: unassigned
+Author:
+  - Aaron Kalin <akalin@martinisoftware.com>
+  - David Aronsohn <wagthattail@me.com>
+Status: Draft
+Type: Standards Track
+---
+
+# Title
+
+Let users specify ohai plugins at bootstrap to provide additional node data on
+the first and subsequent chef runs.
+
+## Motivation
+
+    As a chef user,
+    I want to install ohai plugins during bootstrap,
+    so that I have my ohai facts on my initial chef run.
+
+## Specification
+
+We add a new flag to the knife boostrap subcommand which allows you to specify
+a single file or directory of ohai plugins to be placed in the newly created
+`/etc/client.d` directory upon a bootstrap.
+
+An example command would look like for a single plugin:
+
+`knife bootstrap node.example.com --ohai-plugins="my_custom_plugin.rb" ...`
```

Comment:
Will there be support/is there a use case for a comma separated list? E.g., `--ohai-plugins="my_custom_plugin1.rb,my_custom_plugin2.rb,...,my_custom_pluginN.rb"`